README.md

```diff
@@ -1,137 +1,167 @@
+* Karma's `exclude` option will be used to ignore files in Stryker (using `!` to ignore them)
+* Other karma config will be copied to the `karmaConfig` option in stryker config. These will be used by the `stryker-karma-runner` during mutation testing.
+    * **Note**: Any manual setup you configure in the `karmaConfig` options will *not* be overwritten.
+
+#### Manual setup
+
+**Note**: Using the manual setup will not read your `karma.conf.js` options. You'll probably need to *Override karma config* (see next section)
+
+When the Stryker uses the karma test runner, it uses these default karma config settings:
+
+```javascript
+{
+  browsers: ['PhantomJS'], // *
+  frameworks: ['jasmine'], // *
+  autoWatch: false,
+  singleRun: false,
+}
+// * these settings can be overriden, see next paragraph
+```
+
+For the first test run, stryker enables code coverage. If so, these additional settings are used:
+
+```javascript
+  coverageReporter: {
+    type: 'in-memory' 
+  },
+  preprocessors: {
+    'src/file1.js': 'coverage', // for each file to be mutated
+    'src/file2.js': 'coverage'
+  },
+  plugins: ['karma-coverage'] // or left blank (by default, karma loads all plugins starting with karma-*
+```
+
+#### Override karma config
+
+You can override karma config by adding a `karmaConfig` property to your `stryker.conf.js` file.
+
+For example:
+
+```javascript
+karmaConfig: {
+    browsers: ['Chrome'],
+    frameworks: ['mocha']
+}
+```
+
+*Note*: Whichever testFramework you use should also be reflected in the `testFramework` property of stryker itself. For example: `testFramework: 'mocha'`  
+
+Not all karma config can be overriden, as Stryker requires specific functionality from the testRunner to do its magic. 
+Karma config that *cannot* be overriden:
+
+* `files`: The karma-runner will fill this based on the `files` and `filesToMutate` configuration in the `stryker-conf.js` file.
+* `coverageReporter`: This will be enabled for the initial test run, disabled for testing the mutants.
+* `autoWatch`, `singleRun`: Stryker needs full control on when to run the karma tests
+
+### Full config example
+
+```javascript
+// stryker.conf.js
+exports = function(config){
+    config.set({
+        // ...
+        testRunner: 'karma',
+        testFramework: 'jasmine',
+        karmaConfig: { // these are the defaults
+            browsers: ['PhantomJS'],
+            frameworks: ['jasmine'],
+            autoWatch: false,
+            singleRun: false
+        },
+        plugins: ['stryker-karma-runner'] // Or leave out the plugin list entirely to load all stryker-* plugins directly
+        // ...
+    });
+}
+```
+
+## Usage
+
+Use Stryker as you normally would.
+See [http://stryker-mutator.github.io](http://stryker-mutator.github.io) for more info. 
+
+### Debugging
+
+By default, karma logging will be swallowed. 

```

Comment:
As we' re running karma in it's own process, its logging output will be consumed by us.